src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractLoader.java

```diff
@@ -49,12 +49,6 @@
  */
 public abstract class AbstractLoader
     extends DefaultHandler {
-    /** Feature that enables loading external DTD when loading XML files. */
-    private static final String LOAD_EXTERNAL_DTD =
-        "http://apache.org/xml/features/nonvalidating/load-external-dtd";
```

Comment:
please try to do this by means of inner class in AbstractLoader, if it does not help ... ok to make separate top level class `FeauresForVerySecureJavaInstallations` in javadoc make it clear that it is result of conflict of pitest and user requirements.